integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -98,6 +103,10 @@ public String getFieldValue(CreateFileEntry o) {
       = new IndexedSet<>(ID_INDEX, PATH_INDEX);
   private final boolean mIsUserGroupTranslation;
 
+  // Map for holding the async releasing entries for proper umount
+  private final Map<Long, FileInStream> mReleasingReadEntries = new ConcurrentHashMap<>();
```

Comment:
Do we need to check `FileInStream` in release function? For in stream, the file should ready in `completed` status and no need extra action in server side

Reply:
The fileInStream also needs to be closed to make sure resources are released, e.g. the opened RandomAccessFile and the FileChannel used for reading from local worker storage..

Reply:
Thanks. I will make my question more clearly. What I though is:
1. For writing operations, when close the `FileOutStream`. We should wait for the operation completed because  we need to make sure the resource is server (worker or master) side is released. Otherwise, file will be in an `uncompleted` status
2. For reading operations, the `FileInStream` only hold the resources in client side. After umount is called and fuse daemon is down. All resources used in client side will be released, so we don' t need to release the resource explicitly here. (Maybe server keeps grpc connection with the client, but the resource will be released after reach the timeout)